contracts/DelFiPrice.sol

```diff
@@ -27,17 +45,85 @@ contract DelFiPrice is OpenOracleView {
     /// @notice The mapping of medianized prices per symbol
     mapping(string => uint64) public prices;
 
+    /// @notice The binary representation for 'ETH' symbol , used for string comparison
+    bytes32 constant symbolEth = keccak256(abi.encodePacked("ETH"));
+
+    /// @notice The binary representation for 'USDC' symbol, used for string comparison
+    bytes32 constant symbolUsdc = keccak256(abi.encodePacked("USDC"));
+
+    /// @notice The binary representation for 'DAI' symbol, used for string comparison
+    bytes32 constant symbolDai = keccak256(abi.encodePacked("DAI"));
+
+    /// @notice The binary representation for 'REP' symbol, used for string comparison
+    bytes32 constant symbolRep = keccak256(abi.encodePacked("REP"));
+
+    /// @notice The binary representation for 'BTC' symbol, used for string comparison
+    bytes32 constant symbolWbtc = keccak256(abi.encodePacked("BTC"));
+
+    /// @notice The binary representation for 'BAT' symbol, used for string comparison
+    bytes32 constant symbolBat = keccak256(abi.encodePacked("BAT"));
+
+    /// @notice The binary representation for 'ZRX' symbol, used for string comparison
+    bytes32 constant symbolZrx = keccak256(abi.encodePacked("ZRX"));
+
+    /// @notice The binary representation for 'SAI' symbol, used for string comparison
+    bytes32 constant symbolSai = keccak256(abi.encodePacked("SAI"));
```

Comment:
we won't actually need SAI as a string, ( ever probably ) and actually will only need BTC and ETH until coinbase starts to add more assets

Reply:
Good point

Reply:
deleted it in the last commit